README.md

```diff
@@ -88,6 +92,34 @@ corr_matrix(results,col = 'infer.value',isig = TRUE)
 # attr(,"class")
 # [1] "cmatrix" "matrix" 
 ```
+Now, we can clustering the data set variables through ACCA and the correlation matrix.
```

Comment:
I think the right way to write this sentence is:
Now, we can cluster the data...